.travis.yml

```diff
@@ -13,15 +13,15 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL PHPUNIT_TEST=1 RECIPE_VERSION="^1 --prefer-lowest"
-    - php: 7.0
-      env: DB=MYSQL PHPUNIT_TEST=1 RECIPE_VERSION="^1"
+      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
     - php: 7.1
-      env: DB=PGSQL PHPUNIT_TEST=1 RECIPE_VERSION="^4 --prefer-lowest"
```

Comment:
The idea here was to test the module with the minimal dependency version it ostensibly support.

I think, it would be a good idea if all our builds did something like that. But a reasonable argument could be made that it's more important to make our builds as uniform as possible.